client/src/components/Draftail/Draftail.scss

```diff
@@ -192,6 +192,8 @@ $draftail-editor-font-family: $font-sans;
 
 .Draftail-ComboBox__menu {
   padding: 0 theme('spacing.4');
+  padding-top: 5px;
```

Comment:
Please use the theme values. 
Also if we are changing padding top/bottom - we should just update the line above to have the padding.